docs/30_data_standards.md

```diff
@@ -0,0 +1,30 @@
+## Data standards
+DANDI requires uploaded data to adhere to community data standards. 
+These standards help data curators package all of the necessary metadata, and provide a uniform
+structure so that data can be more easily understood and reused by future users. 
+DANDI also leverages these standards to provide features like data validation and 
+automatic metadata extraction and search. DANDI currently supports two data standards: 
+* For cellular neurophysiology, such as electrophysiology and optical physiology, use [Neurodata Without Borders (NWB)](https://www.nwb.org/nwb-neurophysiology/)
+* For neuroimaging data, such as fMRI and PET, use [Brain Imaging Data Structure (BIDS)](https://bids.neuroimaging.io/)
+
+To share data on DANDI, you will first need to convert your data to the 
```

Comment:
not a native speaker here, but since could be one or another, may be
```suggestion
To share data on DANDI, you will first need to convert your data to an 
```
would be more correct?